app/frontend/src/application/hiring_staff/feedbackForm.js

```diff
@@ -29,26 +29,28 @@
 const FORM_ELEMENT_ERROR_CLASS = 'govuk-input--error';
 
 document.addEventListener('DOMContentLoaded', () => {
-  Array.from(document.getElementsByClassName('vacancy-feedback__form')).forEach((form) => {
-    form.addEventListener('submit', (event) => {
-      Array.from(document.querySelectorAll(`select[form="${event.target.id}"]`)).forEach((select) => {
-        if (select.value === '') {
-          if (!formElementHasError(select)) {
-            formElementAddError(select, event.target.dataset.optionNotSelectedMessage);
+  if (document.querySelector('.expired__vacancy-form')) {
```

Comment:
is this actually part of this task or is it aa oversight by me on changes i made earlier in week?